dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -891,6 +895,11 @@
             "modelAsString": true
           }
         },
+        "confidenceScore": {
+          "type": "number",
```

Comment:
Actually, can you add `"x-ms-client-name": "confidence"` as we did in other swaggers? The SDK archboard didn't like "Score" on the properties.

Reply:
I was keeping it consistent with the confidenceScore field in entities. Is the suggestion also to change that field in this case?
Also - should x-ms-client-name only be used for the confidence score fields?

Reply:
@aurghob `x-ms-client-name` just tells the SDK tools to name this field differently when generating the SDKs. The name returned from the service should still be `confidenceScore` for consistency.